### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Ngufor-emmanuella**
**Add Contact form**

- Add the name field that  accepts up to 30 characters.
-  Add the email field that accepts only emails.
- Add the text area that accepts up to 500 characters
- All fields have placeholder text.
- A user cannot submit the form if one or more of the fields is empty.
- Collect data by using the [Formspree service](https://formspree.io/html).
- Add Interactions on the submit button

**Update Readme file**

- Add authors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -182,5 +185,18 @@ <h3>LET'S CONNECT</h3>
...
+                    <h1>Contact me</h1>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a 
+                        project that needs coding. I'd love to help with it </p>
+                        <nav>
+                        <form action="https://formspree.io/f/xrgjovwd" method="post">
+                            <input type="text" id="name" placeholder="Yeramias" maxlength="30" required>
+                            <input type="email" id="mail" name="email address" placeholder="natajayarijl4@gmail.com" required>
+                            <textarea id="msg" rows="5" name="user-message" placeholder="write your message here" maxlength="500" required></textarea>
+                            <button type="submit">Get It Touch </button>
+                        </form>
```
Comment (danmainah) [inline index.html]: - [ ] Kindly make sure to use flexbox to style your form elements, this is a project requirement.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Ngufor-emmanuella, @AbbyNyakara  and @Bornittah, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good PR and Description


# Required Changes ♻️

- [ ] Great job team especially with the design, there is a small error in your current form kindly check the comparison below and fix it.

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image | image

- [ ] Kindly make sure the hover effect in your buttons follow the design given, below is the colors in the design given:
image




_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -182,5 +184,18 @@ <h3>LET'S CONNECT</h3>
...
+                    <h1>Contact me</h1>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a 
+                        project that needs coding. I'd love to help with it </p>
+                        <nav>
+                        <form action="https://formspree.io/f/xrgjovwd" method="post">
+                            <input type="text" id="name" placeholder="Yeramias" maxlength="30" required>
+                            <input type="email" id="mail" name="email address" placeholder="natajayarijl4@gmail.com" required>
+                            <textarea id="msg" rows="7" name="user-message" placeholder="write your message here" maxlength="500" required></textarea>
+                            <button type="submit">Get It Touch </button>
+                        </form>
```
Comment (codepantha) [inline index.html]: - [ ] As requested by the previous reviewer, please implement the hover state for the button as shown below 👇 
image

Comment (codepantha) [inline index.html]: - [ ] As requested by the previous reviewer, kindly make sure to use flexbox to style your form elements, this is a project requirement 🙏

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @Ngufor-emmanuella @AbbyNyakara and @Bornittah  👋, 

## STATUS: REQUESTED CHANGES ♻️ 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (sja-thedude) [reply index.html]: Good job in implementing the changes 👍

Comment (sja-thedude) [reply index.html]: Good job in implementing the changes 👍

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @Ngufor-emmanuella, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**